public/mall/index.php

```diff
@@ -13,332 +19,137 @@
+
+            <div class="flex-container flex-justify-content-space-between flex-align-items-center flex-wrap">
+                <?php
+                    
+                    // Loop counting the number of products until reaching 10 items
+                    // Get a query containing the product's ID for dynamic product detail can work properly
+                    $display_count = 0;
+                    $featured_mall_products = check_featured_mall_products($products);
+                    while ($display_count < FEATURED_PRODUCTS_DISPLAY_NUM) {
+                        echo "<div class='product-card'>
+                        <a href='" . url_for("/store/store-template/product-detail?id={$products[$display_count]['id']}") . "'><img alt='image of a product' src='../media/image/placeholder_262x250.png'></a>
+                        <div class='product-card-details'>
+                          <a class='product-card-title' href='" . url_for("/store/store-template/product-detail?id={$products[$display_count]['id']}") . "'>" . $featured_mall_products[$display_count]["name"] . "</a>
+                          <a class='product-card-shop' href='" . url_for("/store/store-template?id={$products[$display_count]['store_id']}") . "'>" . get_store_name((int) $featured_mall_products[$display_count]["store_id"], $stores) . "</a>
+                          <p class='product-card-price'>$" . $featured_mall_products[$display_count]["price"] . "</p>
+                          <div class='product-card-sale-card'>" . date("Y年m月d日", strtotime($featured_mall_products[$display_count]["created_time"])) . "</div>
+                        </div>
+                      </div>";
+                        $display_count++;
+                    }
+                
+                ?>
+            </div>
+
+            <button class="cards-load-more-bttn" id="load-more-featured-products" onclick="">LOAD MORE</button>
+        </section>
+
+        <section class="mall-home-section" id="featured-stores">
+            <h1>FEATURED STORES<span class="mall-home-section-heading-link"><a href="<?=url_for("/mall/browse/by-store/featured.php");?>">ALL</a></span></h1>
+
+            <div class="flex-container flex-justify-content-space-between flex-align-items-center flex-wrap">
+                <?php
+                    
+                    // Loop counting the number of stores until reaching 10 items
+                    // Get a query containing the store's ID for dynamic store home can work properly
+                    $display_count = 0;
+                    $featured_mall_stores = check_featured_mall_stores($stores);
+                    while ($display_count < FEATURED_STORES_DISPLAY_NUM) {
+                        echo "<div class='store-card'>
+                        <a href='" . url_for("/store/store-template") . "'><img class='store-card-thumbnail' alt='image representation of a shop' src='../media/image/placeholder_262x250.png'></a>
```

Comment:
- Can you use the profile placeholder 143x143 here? I think it would fit more with the store cards?